echoregions/lines/lines_mask.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
+import xarray as xr
+from xarray import DataArray
+
+from .lines import Lines
+
+
+def lines_mask(
+    da_Sv: DataArray, lines: Lines, method: str = "nearest", limit_area: str = None
+):
```

Comment:
Similar to the regions2d case, we want this to be a method `Lines.mask` and not as a separate function.